openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -2241,7 +2243,12 @@ def find_matches(self, entity):
         #  Should we reduce its scope and have a check here to make sure entity is a Topology?
         return self._find_matches(entity)
```

Comment:
(blocking) Do we need to thread the `unique` kwarg through here? Ditto for all other instances of this pattern in the other ParameterHandlers.

Reply:
Yes, we do. I got wrapped up in out-of-scope details when thinking about how to wire it through. I think the argument is passed through each handler now; please verify that this is what you had in mind.